### ISSUE TITLE
Gutenberg: iOS browser font size selector does not work

### ISSUE THREAD

**Issue by rezzap** [Automattic/wp-calypso#29803]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/block-editor/post/ in a mobile browser (tested this on iOS, but user reporting is working on Android)
2. Add a new paragraph block to a post. 
3. Click on 'block settings' to open the font-size options to select through preset font size (small, medium, large)
4. Nothing happens.

#### What I expected
I expected a menu to open like in the desktop browser to select different options.

<img width="421" alt="image 2018-12-29 at 3 44 08 pm" src="https://user-images.githubusercontent.com/10325627/50539938-aac0eb00-0b80-11e9-909e-e01059f16f26.png">

#### What happened instead

Nothing happens and it is not possible to select a different font size. 

![img_c74feef357c5-1](https://user-images.githubusercontent.com/10325627/50539950-c5935f80-0b80-11e9-9d8d-e06a5a0c4848.jpeg)

You can select the font-size number and reset button, but the preset values doesn't work. 

#### Browser / OS version

Replicated the issue in both Chrome (71.0.3578.89) and Safari iOS12.1.2 on iPhone X

#### Context / Source
#user-report in zen-1672232

**Comment by arunsathiya** [comment]
I tested this on an Android smartphone, and I am not able to get that selector to work. 

I also tried reducing the viewport on Google Chrome Version 71.0.3578.98 (Official Build) (64-bit) for macOS 10.14.2 - Mac OS Mojave, and I see the same behaviour. It works when I launch the browser console though using `Option + Command + J` - unsure why it works only for this stage.

Moreover, this seems to be a core Gutenberg issue as I am able to reproduce this on a core install. Leaving this issue open for an additional check from someone else.
